agent/lib/ipaddr_helpers.rb

```diff
@@ -0,0 +1,48 @@
+class IPAddr
```

Comment:
Should I fix all the server and agent stuff to start using `ipaddress` in this PR, or later?